### DISCUSSION TITLE
Eventuelle Verbesserungen !?

### DISCUSSION THREAD

**Discussion by karlo62** [matlen67/soyosource-powercontroller#5] [General]
Ich habe 3 Soyosource und kann dies auch im Teileroutput angeben. Ich kann also 3 x 900 Watt einspeisen. Also 2700 Watt. Ich kann aber Max Output nur bis 2000 Watt eingeben. Wäre es nicht Sinnvoll z.B. Max Output 900 W mit dem Teileroutput zu multiplizieren.
Ist nur ein Gedanke / Vorschlag .
MFG

**Comment by karlo62** [comment]
Habe es hinbekommen den Wert für den Output zu erhöhen. 
Danke !!

**Comment by Jarnsen** [comment]
@karlo62 hast du an jedem Soyo einen ESP oder einen ESP für alle 3?

**Comment by karlo62** [comment]
Ich habe einen ESP8266 für drei Soyosource. Dafür ist ja der Teiler da.

Jarnsen ***@***.***> schrieb am Sa., 10. Aug. 2024, 09:38:

**Reply by Jarnsen** [reply]
ah ok, nutzt du Home Assistant??

**Reply by karlo62** [reply]
Nein, ich habe mal damit rumgedaddelt.
Das ist mir aber zu kompliziert. Der ESP macht eine astreine Nulleinspeisung und mehr brauche ich nicht.

**Reply by Jarnsen** [reply]
ah ok, dachte nur wer mit sowas anfängt hat auch ein SmartHome System. Viel Spaß
